src/python/BFinitPassthrough.py

```diff
@@ -92,7 +92,7 @@ def bf_passthrough_init(port, requestedBaudrate, half_duplex=False):
             if SCRIPT_DEBUG:
                 dbg_print("  '%s'" % line)
             config = re.search('serial ([0-9]+) ([0-9]+) ', line)
-            if config and config.group(2) == "64":
+            if config and (config.group(2) == "64" or config.group(2) == "131136"):
```

Comment:
I had to change this to 
```python
if config and (int(config.group(2)) & 64 == 64):
```
because with BF 4.4 the serial value was `131137`, so it's probably better to just check the the bit enabling the serial port as used for the receiver is better

Reply:
Can you share your bf config? I implemented this feature in bf 4.4 and never seen this value. When the receiver and msp vtx is set it must be 131136 as there should be only two bits set. I will change the code but there might be something funky going on with your setup. It seems like you enabled both MSP and RX serial on the same UART which is a mistake.

Reply:
I just flashed 4.4 to the board. I wasn't sure which board to choose, so I picked the CRAZYBEEF4SX1280 (I know it doesn't do SPI).
Then selected RX on UART1 and also MSP+VTX and in the diff it was 131137
1. If I just had SerialRX selected then is 64 as expected
2. If I select MSP it's 1
3. If I select just VTX (MSP+Displayport) it automatically toggles the MSP select as well (I guess thats where the 1 is coming from)

My configurator is 10.9.0 (697943b)